ddtrace/tracer/option.go

```diff
@@ -102,6 +102,34 @@ func defaults(c *config) {
 			log.Warn("unable to look up hostname: %v", err)
 		}
 	}
+
+	if os.Getenv("DD_TRACE_GLOBAL_TAGS") != "" {
+
+		if c.globalTags == nil {
+			c.globalTags = make(map[string]interface{})
+		}
+
+		osTags := os.Getenv("DD_TRACE_GLOBAL_TAGS")
+		if osTags != "" {
+			for _, tag := range strings.Split(osTags, ",") {
+				if tag != "" {
+					tag = strings.TrimSpace(tag)
+					var keyVal = strings.Split(tag, ":")
+					if len(keyVal) == 2 {
+						var key = strings.TrimSpace(keyVal[0])
+						var val = strings.TrimSpace(keyVal[1])
+						c.globalTags[key] = val
+					}
```

Comment:
It's fine to have a tag without the value, so we need to do:
```go
k := strings.TrimSpace(kv[0])
switch len(kv) {
    case 1:
        WithGlobalTag(k, "")(c)
    case 2:
        WithGlobalTag(k, strings.TrimSpace(kv[1]))(c)
}
```
Also use `WithGlobalTag`